src/core/sys/posix/iconv.d

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+
+    D binding for the POSIX iconv library.
+
+    See_Also:
+       http://pubs.opengroup.org/onlinepubs/009695399/functions/iconv_open.html
+
+*******************************************************************************/
+
+module core.sys.posix.iconv;
+
+version (Posix):
+extern (C):
```

Comment:
About `pure`, I don't _think_ this qualifies, as it is changing the global state via  `errno`, which is not passed via parameters. Is my reasoning correct, or am I missing something?